code/modules/admin/verbs/possess.dm

```diff
@@ -25,11 +25,17 @@
 	usr.control_object = O
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Possess Object") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-/proc/release()
+/proc/release(obj/O in world) //yogs
 	set name = "Release Obj"
 	set category = "Object"
 	//usr.loc = get_turf(usr)
 
+//YOGS start
+	if(!usr.control_object)
+		to_chat(usr, "You need to possess an object first!")
+		return
+//YOGS end

```

Comment:
```suggestion
// yogs end
```

Reply:
Is the comment required to be case sensitive or not? Asking for a friend

Reply:
also asking for a friend